builder/azure/TODO.md

```diff
@@ -1,16 +0,0 @@
-Here's a list of things we like to get done in no particular order:
```

Comment:
This file is 6 years out of date, and does not accurately future prioritized work on the plugin